crates/sozo/ops/src/migration/migrate.rs

```diff
@@ -882,6 +874,89 @@ pub async fn update_manifests_and_abis(
     Ok(())
 }
 
+pub async fn find_authorization_diff<A>(
+    ui: &Ui,
+    world: &WorldContract<A>,
+    diff: &WorldDiff,
+    migration_output: Option<&MigrationOutput>,
+    default_namespace: &str,
+) -> Result<(Vec<ResourceWriter>, Vec<ResourceWriter>)>
+where
+    A: ConnectedAccount + Sync + Send,
+    <A as Account>::SignError: 'static,
+{
+    let mut grant = vec![];
+    let mut revoke = vec![];
+
+    for c in &diff.contracts {
+        // remote is none meants its not deployed.
+        // now it could have been deployed during this run
+        // which can be checked from migration_output
+        // if c.remote_class_hash.is_none() {
+        //     continue;
+        // }
```

Comment:
If the check is no longer relevant, we may remove the check code, and only specify why we iterate without checking the remote class hash.